### ISSUE TITLE
Manifest merger not working?

### ISSUE THREAD

**Issue by hameno** [simpligility/android-maven-plugin#715]
Hello,
I'm trying to build an app which includes the google play services 8.4.0. 
In the AndroidManifest.xml of play-services-measurement there is a provider

```
  <provider
      android:authorities="${applicationId}.google_measurement_service"
      android:name="com.google.android.gms.measurement.AppMeasurementContentProvider"
      android:exported="false"/>
```

This causes an error when it gets merged into my apps AndroidManifest.xml.
When I try to replace the value like this (as mentioned in https://code.google.com/p/analytics-issues/issues/detail?id=784)

```
<provider
            tools:replace="android:authorities"
            android:name="com.google.android.gms.measurement.AppMeasurementContentProvider"
            android:authorities="com.example.app.google_measurement_service"
            android:exported="false" />
```

it still fails with the following error

```
Error: [/var/lib/jenkins/workspace/app1/app1/target/AndroidManifest.xml:32, /var/lib/jenkins/workspace/app1/app1/target/unpacked-libs/cgag_play-services-measurement_8.4.0/AndroidManifest.xml:12] Trying to merge incompatible /manifest/application/provider[@name=com.google.android.gms.measurement.AppMeasurementContentProvider] element:
  <provider
--    @android:authorities="com.example.app.google_measurement_service"
  <provider
++    @android:authorities="${applicationId}.google_measurement_service"
```

Are these markers even supported?

**Comment by hameno** [comment]
Okay, found it, I had to diable the old manifest-update with mergeManifests as it would run automatically before the real manifest-merger goal after all dependencies have been extracted. Should really be in the docs

**Comment by mosabua** [comment]
Hm... can you clarify or submit a Javadoc or asciidoc PR with the explanation?

**Comment by hameno** [comment]
well, basicly: old merger is deprecated but still enabled by default, wouldn't it be better to disable it and let the new one be the default?

**Comment by mosabua** [comment]
At this stage probably yes... if we do that we just ref to next minor version number for the plugin and reference it in the changelog.. want to do that in a PR? Also wdyt @simpligility/android-maven-plugins-core-committers

**Comment by Shusshu** [comment]
Yes it should be the default

**Comment by hameno** [comment]
Should be resolved with #757

**Comment by william-ferguson-au** [comment]
Closed with #757
